fhem/FHEM/lib/Device/MySensors/Constants.pm

```diff
@@ -133,11 +146,23 @@ use constant {
   S_CUSTOM                => 23,
   S_DUST                  => 24,
   S_SCENE_CONTROLLER      => 25,
+  S_RGB_LIGHT             => 26,
+  S_RGBW_LIGHT            => 27,
+  S_COLOR_SENSOR          => 28,
+  S_HVAC                  => 29,
+  S_MULTIMETER            => 30,
+  S_SPRINKLER             => 31,
+  S_WATER_LEAK            => 32,	
+  S_SOUND                 => 33,	
+  S_VIBRATION             => 34,
+  S_MOISTURE              => 35,
 };
 
-use constant sensorTypes => qw{ S_DOOR S_MOTION S_SMOKE S_LIGHT S_DIMMER S_COVER S_TEMP S_HUM S_BARO S_WIND
+use constant sensorTypes => qw{ S_DOOR S_MOTION S_SMOKE S_LIGHT S_BINARY S_DIMMER S_COVER S_TEMP S_HUM S_BARO S_WIND
```

Comment:
I had to remove S_LIGHT as it increases the index of all following entries. As the index is used to directly translate in between numerical message values and sensortypes having aliases ain't work.
https://github.com/ntruchsess/fhem-mirror/commit/37a5dcf89463c5b686a09bedcc9e46f4cfc9621f#diff-9ab2799f37fc2b24a1e799fb6d487216L159